lander:
  ros__parameters:
    target_local_position: [0.0, 0.0, 0.0] 
    target_seek_altitude: 12.0
    approach_holddown: 5
    approach_speed: 1.0
    approach_radius: 1.5
    max_yaw_speed: 0.25
    camera_yaw_correction: 0.785398163
    camera_vertical_fov: 0.851720675
    max_speed_xy: 1.0
    max_accel_xy: 0.1
    descend_radius: 0.33
    descend_max_speed_xy: 0.1
    descend_holddown: 3
    max_speed_z: 0.5
    max_accel_z: 0.05
    land_max_speed_xy: 0.05
    land_altitude: 1.0
